content/docs/examples/combination/pathwithother/content.md

```diff
@@ -0,0 +1,40 @@
++++
+fragment = "content"
+weight = 200
+
+title = "Path with Other"
+
+[sidebar]
+  enable = true
++++
+
+The `path` type can also be used with other types like `dockerv2` or `host`.
+This feature enables you to redirect requests based on their path to different
+endpoints. For example, we use this feature to redirect create vanity URLs for
+our packages. Take a look at this example:
+
+```
+pkg.example.com                     3600 IN A      127.0.0.1
+_redirect.pkg.example.com           3600 IN TXT    "v=txtv0;type=path"
+_redirect.foo.pkg.example.com       3600 IN TXT    "v=txtv0;type=gometa;to=https://github.com/foo/foo"
+_redirect.bar.pkg.example.com       3600 IN TXT    "v=txtv0;type=gometa;to=https://github.com/foo/bar"
+_redirect.baz.pkg.example.com       3600 IN TXT    "v=txtv0;type=dockerv2;to=https://gcr.io/foo/baz"
```

Comment:
```suggestion
_redirect.baz.pkg.example.com       3600 IN TXT    "v=txtv0;type=host;to=https://github.com/foo/baz/blob/master/README.md#deprecated"
```

Reply:
Using host as an additional example seems a bit simpler. Especially considering our current stability of dockerv2 :)